specification/schemas/BaseCustoms.json

```
@@ -35,6 +35,23 @@
       "example": "DAP",
       "description": "DAP (Delivered At Place) or DDP (Delivered Duty Paid)."
     },
+    "shipping_value": {
+      "oneOf": [
+        {
+          "allOf": [
+            {
+              "$ref": "#/components/schemas/Price"
+            },
+            {
+              "description": "The cost of shipping for this shipment."
```

```suggestion
              "description": "The shipping cost paid by the customer, for tax and duty purposes."
```
Let's indicate that these are the costs charged to the end user, so it's not confused with the shipment price or total value.